### DISCUSSION TITLE
Relationships

### DISCUSSION THREAD

**Discussion by lazerg** [spatie/laravel-deleted-models#11] [Q&A] [answered]
Hello. Thank you for the package; it is extremely useful. I was wondering what if I have a `users` table with its `posts`, and while deleting user I also want to delete the user's posts as well? Do you currently have such functionality?

**Comment by freekmurze** [comment, marked as answer]
No, the package doesn't handle that. You must handle this yourself. If you want the posts to go in the `deleted_models` table, you need to manually delete those posts before deleting the user.

**Comment by inmanturbo** [comment]
It would be pretty amazing if the entire relationship tree could be serialized under the deleted model in cascading fashion. Of course if there where a lot of related models that would be a very large blob.
